react/features/base/label/components/Label.web.js

```diff
@@ -11,47 +14,111 @@ import AbstractLabel, {
 type Props = AbstractProps & {
 
     /**
-     * Additional CSS class names to add to the root of {@code Label}.
+     * An object containing the CSS classes.
      */
-    className: string,
+    classes: Object,
+
+    /**
+     * The color of the label.
+     */
+    color: string,
+
 
     /**
      * HTML ID attribute to add to the root of {@code Label}.
      */
-    id: string
+    id: string,
+
+    /**
+     * Click handler if any.
+     */
+    onClick?: Function,
+
+};
 
+/**
+ * Creates the styles for the component.
+ *
+ * @param {Object} theme - The current UI theme.
+ *
+ * @returns {Object}
+ */
+const styles = theme => {
+    return {
+        label: {
+            ...withPixelLineHeight(theme.typography.labelRegular),
+
+            alignItems: 'center',
+            background: theme.palette.ui04,
+            borderRadius: theme.shape.borderRadius / 2,
+            color: theme.palette.text01,
+            display: 'flex',
+            height: 28,
+            margin: '0 0 4px 4px',
+            padding: '0 8px'
+        },
+        withIcon: {
+            marginLeft: 8
+        },
+        clickable: {
+            cursor: 'pointer'
+        },
+        [COLORS.white]: {
+            background: theme.palette.text01,
+            color: theme.palette.ui04,
+
+            '& svg': {
+                fill: theme.palette.ui04
+            }
+        },
+        [COLORS.green]: {
+            background: '#31B76A'
```

Comment:
Shouldn't this be tokenized?

Reply:
I could not find this exact color. But I'll try to  use some similar colors from our token system.